apps/base/utils.py

```diff
@@ -69,3 +74,23 @@ def decode_data(data):
     for i in data:
         decoded.append(base64.decodestring(i+"=="))
     return decoded
+
+
+def send_slack_notification(webhook=settings.SLACK_WEBHOOKS['default'], message=""):
+    '''Send slack notification to any workspace
+
+    Keyword Arguments:
+        webhook {string} -- slack webhook URL (default: {settings.SLACK_WEBHOOKS['default']})
+        message {str} -- JSON/Text message to be sent to slack (default: {""})
+    '''
+    if settings.DEBUG is False:
+        try:
+            response = requests.post(
+                webhook,
+                data=json.dumps({'text': str(message)}),
+                headers={'Content-Type': 'application/json'}
+            )
+            return response
+        except Exception as e:
```

Comment:
Instead of doing exception handling, it would make more sense to check based on the response status.

Reply:
I think it should just work since Travis is also just a machine running all the tests. It should ideally work on all the machines. Can you please spend some more time on this to figure it out? Thanks for your patience.

Reply:
Fixed this. I missed adding in mocks for test cases where send_slack_notifications was indirectly used